docs/naisjob/README.md

```diff
@@ -5,7 +5,7 @@ description: This document describes the properties a Naisjob should have.
 # Naisjob
 A Naisjob is very similar to a [NAIS Application](../nais-application/README.md), just that it only runs once.
 It is built on Kubernetes [Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) and [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/).
-In short a Job as opposed to Application will just run a pod one time. 
+Compared to an Application, a Job will only start a pod once. 
```

Comment:
```suggestion
Compared to an Application, a `Job` will only start a pod once (unless it fails, usual retry-mechanics hold for `Job`s too).
```

Reply:
Hvis vi skal ha med `unless... ` så syns jeg den burde presis for jobs. Enten lenke til doc, eller ha mer info etter på.